dom/base/nsContentUtils.cpp

```diff
@@ -7760,6 +7760,13 @@ nsContentUtils::SendMouseEvent(nsCOMPtr<nsIPresShell> aPresShell,
     msg = eMouseUp;
   } else if (aType.EqualsLiteral("mousemove")) {
     msg = eMouseMove;
+
+    // If need to draw mouse cursor, set point then force composition
+    if(widget->GetDrawMouse()) {
+      ScreenIntPoint point(floor(aX+0.5), floor(aY+0.5));
+      widget->SetScreenIntPoint(point);
+      widget->ForceComposition();
```

Comment:
This code block will be hit even |mDrawVirtualMouse| is false, screenIntPoint in nsWindow wil lbe modified either from |nsWindow::NotifyHoverMove| (real mouse input) or |nsContentUtils::SendMouseEvent| here. 
Is it your intention ?